code/ct_flowlog_activator.py

```diff
@@ -335,81 +381,161 @@ def list_stack_instance_by_region(target_session, stack_set_name, region):
         for page in stackset_result:
             if 'Summaries' in page:
                 stackset_list.extend(page['Summaries'])
-        
+
         return stackset_list
-        
+
     except Exception as e:
-        LOGGER.error("List Stack Instance error: %s" % e)
+        LOGGER.error("List Stack Instance error: %s" % e, exc_info=True)
         return []
 
-def lambda_handler(event, context):    
+
+def lambda_handler(event, context):
     LOGGER.info('Lambda Handler - Start')
```

Comment:
@wellsiau-aws Did you check this diff? keen to have your eyes on this bit of code change

Reply:
honestly, I missed it and then later realized it when I ran my test, I'll look at it again

Reply:
found issue with https://github.com/aws-samples/aws-ct-vpc-flowlogs/issues/8, which I will address as separate PR.